easybuild/easyblocks/r/rust.py

```diff
@@ -86,32 +88,31 @@ def install_step(self):
             # make sure that all shared libraries use RPATH, not RUNPATH;
             # cfr. https://github.com/easybuilders/easybuild-easyconfigs/issues/18079
             shlib_ext = get_shared_lib_ext()
-            shared_libs = glob.glob(os.path.join(self.installdir, 'lib', 'lib*.%s' % shlib_ext))
-
-            runpath_regex = re.compile(r"\(RUNPATH\)\s+Library runpath")
+            shared_libs = glob.glob(os.path.join(self.installdir, 'lib', f'lib*.{shlib_ext}'))
+            executables = glob.glob(os.path.join(self.installdir, 'bin', '*'))
+            binary_files = [file for file in executables + shared_libs if is_binary(read_file(file, mode='rb'))]
```

Comment:
I prefer doing this only when required, i.e. only for Rust > 1.79.0

I was working on the exact same fix before I noticed you had this PR open already.

I'll push in an extra commit to make this version-specific...

Reply:
good point, done in https://github.com/easybuilders/easybuild-easyblocks/pull/3660/commits/d3537127b646232173c6b5268628fbb5eec79a0e